validator/packaging/validator.toml.example

```diff
@@ -29,6 +29,7 @@
 bind = [
   "network:tcp://127.0.0.1:8800",
   "component:tcp://127.0.0.1:4004"
+  "consensus:tcp://127.0.0.1:5050"
```

Comment:
s/5050/5005

Reply:
This is not a typo, the default port is 5050: https://github.com/hyperledger/sawtooth-core/blob/master/validator/sawtooth_validator/config/validator.py#L34